pkgs/initiator/initiator.go

```diff
@@ -563,11 +569,30 @@ func (c *Initiator) StartDKG(id [24]byte, withdraw []byte, ids []uint64, network
 		return nil, nil, err
 	}
 	c.Logger.Info("✅ verified owner and nonce master signature")
-	keyshares, err := GeneratePayload(dkgResults, reconstructedOwnerNonceMasterSig.Serialize())
+	keyshares, err := GeneratePayload(dkgResults, reconstructedOwnerNonceMasterSig.Serialize(), owner, nonce)
 	if err != nil {
 		return nil, nil, err
 	}
 	c.Logger.Info("✅ verified master signature for ssv contract data")
+	// sending back to operators results
+	depositData, err := json.Marshal(depositDataJson)
+	if err != nil {
+		return nil, nil, err
+	}
+	keysharesData, err := json.Marshal(keyshares)
+	if err != nil {
+		return nil, nil, err
+	}
+	resultMsg := &wire.ResultData{
+		Operators:          ops,
+		InitiatorPublicKey: pkBytes,
+		DepositData:        depositData,
+		KeysharesData:      keysharesData,
+	}
+	err = c.sendResult(resultMsg, ops, consts.API_RESULTS_URL, id)
```

Comment:
I don't understand why do we need to send this data back to the operators. they should be able to reproduce this whole thing using info the got at init. this looks like a possible attack vector.

Reply:
we also don't want the operators to be able to use these by themselves. this belong to the initiator only. (the complete deposit and keyshare files)

Reply:
https://github.com/bloxapp/ssv-dkg/pull/29#discussion_r1415948572